core/file/ctime_spec.rb

```diff
@@ -14,6 +14,21 @@
     File.ctime(@file).should be_kind_of(Time)
   end
 
+  platform_is :linux do
+    it "Returns the change time for the named file (the time at which directory information about the file was changed, not the file itself) with microseconds." do
```

Comment:
I fixed it along with a few other cases in c5de60070b5994ecb8b65825fa90d0d82915cc9e